README.md

```diff
@@ -131,7 +131,7 @@ click to enable line selection.
 
 # Documentation and Help
 
-Micro has a built-in help system which you can access by pressing `CtrlE` and typing `help`. Additionally, you can
+Micro has a built-in help system which you can access by pressing `Ctrl-E` and typing `help`. Additionally, you can
```

Comment:
Added a hyphen since the keystrokes that were set in fixed-width in other pages mostly had hyphens.